test/Jobs.js

```diff
@@ -40,7 +39,7 @@ let datasetPid1 = null,
   jobIdGroup4 = null,
   encodedJobIdGroup4 = null,
   jobIdGroup5 = null,
-  encodedJobIdGroup5 = null,  
+  encodedJobIdGroup5 = null,
```

Comment:
Sorry for the long diff. Everything down to like 4160 should be pure whitespaces changes (my editor removes trailing whitespace).

We should update eslint to cover js files to minimize these formatting changes.